config.py

```diff
@@ -10,5 +13,5 @@
 PASSWORD = ""  # redis password
 
 PRIVATE_CHAT_ID = -1001234567890  # CHAT WHERE YOU WANT TO STORE VIDEOS
-COOKIE = ""  # COOKIE FOR AUTHENTICATION (get from chrome dev tools) ex: "PANWEB=1; csrfToken=; lang=en; TSID=; __bid_n=; _ga=; __stripe_mid=; ndus=; browserid==; ndut_fmt=; _ga_06ZNKL8C2E=" (dont use this)
-ADMINS = [1317173146]
\ No newline at end of file
+COOKIE = os.getenv("COOKIE", None)  # COOKIE FOR AUTHENTICATION (get from chrome dev tools) ex: "PANWEB=1; csrfToken=; lang=en; TSID=; __bid_n=; _ga=; __stripe_mid=; ndus=; browserid==; ndut_fmt=; _ga_06ZNKL8C2E=" (dont use this)
```

Comment:
can you like this for all ?
like 

```
COOKIE = os.getenv("COOKIE", "") 
PRIVATE_CHAT_ID = os.getenv("PRIVATE_CHAT_ID ",-1001234567890)
```

Reply:
I have deleted my fork can you provide me collab at repo ?

Reply:
> I have deleted my fork can you provide me collab at repo ?

yup wait